common/array.js

```diff
@@ -17,3 +17,12 @@ export function orderBy(array, property, order) {
     return `${aProperty}`.localeCompare(`${bProperty}`) * (isDesc ? -1 : 1)
   })
 }
+
+export function shuffle(inputArray) {
+  const outputArray = [...inputArray]
+  for (let i = outputArray.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1))
+    ;[outputArray[i], outputArray[j]] = [outputArray[j], outputArray[i]]
```

Comment:
why the `;` at the beginning?`

Reply:
`;` is for separating `[]` with L24, without it, this will become `Math.floor(Math.random() * (i + 1))[]`. Prettier config not allow `;` at the end of each line